tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -55,41 +55,48 @@ def _deterministic_libtool_flags(ctx):
         return ["-D"]
     return []
 
+def _target_os_version(ctx):
+    platform_type = ctx.fragments.apple.single_arch_platform.platform_type
+    xcode_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
+    return xcode_config.minimum_os_for_platform_type(platform_type)
```

Comment:
can this actually be known at the time this rule is configured? I assume the value we want here is actually more like the one you had before, which was replaced at compile time with the one for the current build

Reply:
This is what was used by `xcode_swift_toolchain` before, I think this should return the right one.

Is that possible to expand cc toolchain variable like `%{version_min}` here? I tried but the `%{version_min}` string is passed verbatim if you pass it to a rule attribute.